starlink_grpc.py

```diff
@@ -363,17 +363,8 @@
 import math
 import statistics
 from typing import Any, Dict, Iterable, List, Optional, Sequence, Tuple, get_type_hints
-try:
-    from typing import TypedDict, get_args
-except ImportError:
-    # Python 3.7 does not have TypedDict, so fake it so the run time still
-    # works, even though static type checker probably will not.
-    def TypedDict(name, types):  # pylint: disable=invalid-name
-        return type(name, (dict,), {"__annotations__": types})
-
-    def get_args(tp: Any) -> Tuple[Any, ...]:
-        return tp.__args__
-
+from typing_extensions import TypedDict, get_args
+from xmlrpc.client import Boolean
```

Comment:
What's this line (the `from xmlrpc.client import Boolean`) for?

Reply:
Excellent question, I assume I typed `Boolean` out of habit and that's what was imported. I'll remove it